guide/it/it.po

```diff
@@ -14908,12 +14908,10 @@ msgstr ""
 #: C/gnucash-guide.xml:40(para)
 msgid ""
 "For a complete guide on <guilabel>trading</guilabel> accounts you can take a "
-"look <ulink url=\"&url-selinger;"
-"accounting/tutorial.html\">&tit-sel-tut;</ulink> by Peter Selinger."
+"look &ulink-sel-tut; by Peter Selinger."
 msgstr ""
 "Per una guida completa sui conti per il <guilabel>trading</guilabel> si "
-"consiglia di consultare la guida <ulink url=\"&url-selinger;"
-"accounting/tutorial.html\">&tit-sel-tut;</ulink> Peter Selinger (in inglese)."
+"consiglia di consultare la guida &ulink-sel-tut; Peter Selinger (in inglese)."
```

Comment:
Vor Peter fehlt ein `di `.

Im Guide können wir dann die msgid durc den msgstr ersetzen.

Hintergrund: Wir haben wohl an dem Link schon mal gebastelt ohne zu beachten, daß it po-basiert ist, weshalb der Absatz, da dann fuzzy auf Englisch eingestezt wurde.

Imzwischen haben wir die po-Geschichte abgeklemmt, da das Dokument sonst inzwischen englisch mit italienischen Fragmenten wäre. Also möglichst it/gnucash-guide.xml aktuell halten.

Die Ersetzungen in der po-Datei erleichtern einem möglichen zukünftigen Übersetzer die Arbeit, sind also wünschenswert.